packages/features/eventtypes/components/AddMembersWithSwitch.tsx

```diff
@@ -132,6 +154,19 @@ const AddMembersWithSwitch = ({
         )}
         {!assignAllTeamMembers || !automaticAddAllEnabled ? (
           <CheckedHostField
+            onInputChange={(inputValue) => {
+              setInputValue(inputValue);
+            }}
+            onKeyDown={(e) => {
+              if (e.key === "Enter") {
+                if (setMemberInviteModal && noOptionLeft && handleEmailInvite) {
+                  e.preventDefault();
+                  setMemberInviteModal(true);
+                  setNoOptionLeft(false);
+                  handleEmailInvite(inputValue);
```

Comment:
can we remove spaces from the `inputValue`? The input field itself doesn't allow spaces. When testing I typed `free@example, pro@example.com` which gave me the error 'Enter emails'